carma-messenger-ui/website/main.html

```diff
@@ -144,7 +135,6 @@
                 </div>
             </div>
 
-            <!-- widget area for truck inspection -->
```

Comment:
We should be able to remove the next line `<div id="divWidgetArea" ></div>` as well.

Reply:
Done!